demisto_sdk/commands/coverage_analyze/helpers.py

```diff
@@ -0,0 +1,189 @@
+import io
+import json
+import logging
+import os
+import sqlite3
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import Dict, Iterable, List, Optional
+
+import coverage
+import requests
+
+logger = logging.getLogger('demisto-sdk')
+
+
+def fix_file_path(coverage_file: str, code_file_absolute_path: str):
+    """
+
+    Args:
+        coverage_file: the .coverage file this contains the coverage data in sqlite format.
+        code_file_absolute_path: the real absolute path to the measured code file.
+
+    Notes:
+        the .coverage files contain all the files list with their absolute path.
+        but our tests (pytest step) are running inside a docker container.
+        so we have to change the path to the correct one.
+    """
+    with sqlite3.connect(coverage_file) as sql_connection:
+        cursor = sql_connection.cursor()
+        index = cursor.execute('SELECT count(*) FROM file').fetchall()[0][0]
+        if not index == 1:
+            logger.error('unexpected file list in coverage report')
+        else:
+            cursor.execute('UPDATE file SET path = ? WHERE id = ?', (code_file_absolute_path, 1))
+            sql_connection.commit()
+        cursor.close()
+    if not index == 1:
+        logger.debug(f'removing coverage report for {code_file_absolute_path}')
+        os.remove(coverage_file)
+
+
+def get_coverage_obj(coverage_file: Optional[str], report_dir: Optional[str], load_old: bool = True,
+                     combine_from_content_repo: bool = False, precision: int = 2) -> coverage.Coverage:
+    """
+    Args:
+        coverage_file(str): the coverage file path (the default is .coverage).
+        report_dir(str): the directory where the reports files should be placed.
+        load_old(bool): load the file (instead of overide).
+        combine_from_content_repo(bool): load .coverage files from content repo.
+        percision(int): digits after the decimal point (e.g. for percision=2 74.3586 -> 74.35)
+    """
+    coverage_obj = coverage.Coverage(config_file=False, auto_data=False)
+    coverage_obj.set_option('report:precision', precision)
+
+    if coverage_file:
+        coverage_obj.set_option('run:data_file', coverage_file)
+    if report_dir:
+        coverage_obj.set_option('html:directory', os.path.join(report_dir, 'html'))
+        coverage_obj.set_option('xml:output', os.path.join(report_dir, 'coverage.xml'))
+        coverage_obj.set_option('json:output', os.path.join(report_dir, 'coverage.json'))
+    if load_old:
+        coverage_obj.load()
+    if combine_from_content_repo:
+        coverage_obj.combine(coverage_files())
+    # uncomment the following for debug purposes
+    # self._cov.set_option('json:pretty_print', True)
+
+    return coverage_obj
+
+
+def coverage_files() -> Iterable[str]:
+    """
+    iterate over the '.coverage' files in content repo.
```

Comment:
SDK is not specific for the content repo, only the repo structure (packs folder and its hierarchy)
```suggestion
    iterate over the '.coverage' files in the repo.
```